prow/spyglass/spyglass_test.go

```diff
@@ -1240,22 +1279,22 @@ func TestFetchArtifactsPodLog(t *testing.T) {
 	}
 
 	for _, key := range testKeys {
-		result, err := sg.FetchArtifacts(context.Background(), key, "", 500e6, []string{"build-log.txt"})
+		containers := []string{"test-1", "test-2"}
+		result, err := sg.FetchArtifacts(context.Background(), key, "", 500e6, []string{fmt.Sprintf("%s-%s", containers[0], singleLogName), fmt.Sprintf("%s-%s", containers[1], singleLogName)})
```

Comment:
It looks like the passed string slice is only appropriate for the multiple container case, but this includes tests for the single container case as well.

Reply:
It seemed like this test was just testing that FetchArtifacts() would fetch PodLogArtifacts when the build-logs don't exist in gcs storage.  I kinda assumed that if it worked for multiple containers, then it would work for a single container so I changed it to test just multiple containers.  For completeness, it makes sense that I add back the single container use case.  Additionally, this made me just realize that it would be good to add a testcase for cases with multiple build logs where some build logs are in gcs and some need to be streamed, so I'll go ahead and add that as well.